services/firebase.sample.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
If required I can share the credentials for the DB I've created for this project